probes/external/external.go

```
@@ -119,6 +119,9 @@ func (p *Probe) Init(name string, opts *options.Options) error {
 		p.l = &logger.Logger{}
 	}
 	p.c = c
+	if p.c == nil {
+		p.c = &configpb.ProbeConf{}
+	}
```

Zero config doesn't work for external probe at all. We should return an error if an external probe config is nil.

---

You are right, as `command` is required.
I remove my change on external as it's already handle via `GetCommand()`